src/app/components/TradingChart/index.tsx

```diff
@@ -60,6 +67,8 @@ export function TradingChart(props: ChartContainerProps) {
         autosize: true,
         // toolbar_bg: '#a3a3a3',
         theme: props.theme,
+        has_no_volume: false,
```

Comment:
Let's set this to `true` for any line charts, so that we don't have a non-functional volume indicator showing up. We can re-use the `hasDirectFeed` variable from helpers.tsx

Reply:
It will re-render entire entire trading view chart when symbol is changed (currently it only shows trading views loading indicator when changing).
Remove it anyway?

Reply:
In that case, it's not worth it - thanks for clarifying